src/lib/textile/ThreadDB.js

```diff
@@ -29,7 +31,7 @@ export class ThreadDB {
     return this.db.collection('Profiles')
   }
 
-  constructor(privateKey: TextileCrypto.PrivateKey) {
+  constructor(privateKey: TextileCrypto.PrivateKey, wallet: any) {
```

Comment:
why are we passing wallet? this is dangerous because wallet changes when switching networks. ThreadDB doesnt

Reply:
will check